src/student_success_tool/generation/pdp/raw_course.py

```diff
@@ -150,7 +150,7 @@ def academic_year(self, min_yr: int = 2010, max_yr: t.Optional[int] = None) -> s
             end_date=(date(max_yr, 1, 1) if max_yr else "today"),
         ).replace(day=1)
         end_dt = start_dt.replace(year=start_dt.year + 1)
-        return f"{start_dt:%Y}-{end_dt:%Y}"
+        return f"{start_dt:%Y}-{end_dt:%y}"
```

Comment:
3x logic duplication is right on the cusp of wanting a consolidated function/method. I think it's fine as-is, since we don't expect to ever have, say, 4x duplication?